cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -6,11 +6,14 @@ use self::cincinnati::plugins::prelude_plugin_impl::*;
 
 use crate::conditional_edges::{ConditionalEdge, ConditionalUpdateEdge, ConditionalUpdateRisk};
 use commons::GRAPH_DATA_DIR_PARAM_KEY;
+use std::cmp::Ordering;
 use std::collections::HashSet;
 use std::path::Path;
 
 pub static DEFAULT_KEY_FILTER: &str = "io.openshift.upgrades.graph";
 static SUPPORTED_VERSIONS: &[&str] = &["1.0.0", "1.1.0", "1.2.0"];
```

Comment:
Can we drop this now?  I'd expect the SemVer-based `MAX_SUPPORTED_VERSION` logic to cover this use-case, and having two things that need bumping when we grow additional support seems like more work than shifting entirely to the new flow and only bumping `MAX_SUPPORTED_VERSION`.